vsphere_plugin_common/__init__.py

```diff
@@ -1428,6 +1490,13 @@ def _update_vm(self, server, cdrom_image=None, remove_networks=False):
                     'Edit cdrom from template. '
                     'Template should have no inserted cdroms.')
                 cdrom_attached = True
+                # skip if cdrom is already attached
+                if isinstance(
+                    device.backing, vim.vm.device.VirtualCdrom.IsoBackingInfo
+                ):
+                    if str(device.backing.fileName) == str(cdrom_image):
+                        logger().info("Same cdrom is already attached.")
```

Comment:
This should probably go back to saying "CDROM is already attached." as it did before- 'Same' is decreasing clarity in this case.